new-frontiers-14/frontier-station-14
# Whitelist cmo stamp

## About the PR



Adds  - RubberStampCMO to the whitelist.

## Why / Balance


 - RubberStampCMO is an illegal prototypes despite being a fill for the CMO locker. This whitelists it so that there's no flag.

## How to test


Upload something with the locker and see if it flags. Coincidentally like the stasis shuttle.

## Media



## Breaking changes


None probably.

**Changelog**




- tweak: RubberStampCMO  to whitelist
